config/drupal-8/drupal-8.4-deprecations.yml

```diff
@@ -1,5 +0,0 @@
-# Contains automatic fixes for some deprecations introduced in Drupal 8.4.
```

Comment:
Does this one have a `.php` version?

Reply:
I might have missed that, during release or my for to your master.

Could you add it?